docs/sections/abstract.html

```diff
@@ -1,8 +1,10 @@
 <section id="abstract">
   <p>
     This specification describes a Linked Data vocabulary for asserting
-    Verifiable Credentials related to traceability information, such as chemical
-    properties, mechanical properties, country of origin, and other attributes
-    used to determine the status of a products and materials in a supply chain.
+    Verifiable Credentials related to supply chain traceability information,
+    such as country of origin, chemical properties, mechanical properties, and
+    other attributes used to determine the legitimacy of organizations
+    participating in global trade and the status of products and materials in
+    the supply chain.
```

Comment:
For as long as I've been involved, it has been asserted that this traceability vocab is about more than _supply chain_ traceability.  This edit adjusts the abstract to allow for such broader applicability (though it doesn't delve deeply into that breadth).  Similar edits will be needed in the larger section above, if this abstract edit is acceptable.
```suggestion
    Verifiable Credentials related to supply chain and other traceability 
    information, similar to what is often referred to as "provenance",
    including country of origin, chemical properties, mechanical properties,
    and other attributes of products and materials.  VCs using these terms 
    can then be used to help determine the legitimacy of organizations
    participating in global trade and the status of the products and 
    materials described therein.
```

Reply:
Excellent, @TallTed !